### ISSUE TITLE
Translations for backend

### ISSUE THREAD

**Issue by bastrapp** [ultical/ultical#43]
The (for now) only time we are producing text to the user in the backend is when creating emails. Those, of course, should be easily translatable. The frontend also has such a file structure to support translations.

I see two possible solutions
1. Create another text-file for (backend) translations
   - \+ Clean
   - \- Two files to maintain
2. Always send the customized text with the request to send the mail.
   - \+ Only one translation file
   - \- Seems like a hack

Any ideas?

**Comment by bslbckr** [comment]
I think we should go with the first proposed option. 
@bastrapp with "two files to maintain" you are referring to the files one for backend and one for frontend? The only solution to get rid of this, I currently could imagine, is that the backend is also providing the translations for the frontend. This however would generate a huge amount of additional requests and probably slows down the frontend :(

**Comment by bastrapp** [comment]
Yes, you guessed right, I am referring to a backend as well as a frontend translation file. Seems like we are sticking with this...
